src/resources/MachineLearning/ModelListing/ModelListingConfiguration.ts

```diff
@@ -0,0 +1,24 @@
+export interface MLListingModel {
+    modelDisplayName: string;
+    modelId: string;
+    engineId: string;
+    estimatedPreviousModelUpdateTime: number;
+    nextModelUpdateTime: number;
+    platformVersion: 1 | 2;
+    modelSizeStatistic: number;
+    readyForAssociation: boolean;
```

Comment:
Update this property before open to review. Also need to add customer error field